lib/prediction/predictor.ts

```diff
@@ -90,34 +94,46 @@ class Worker extends events.EventEmitter {
     }
 
     start() {
-        const args = [
-            'server',
-            '--stdin',
-            '--path', this._modeldir,
-        ];
-        if (process.env.GENIENLP_EMBEDDINGS)
-            args.push('--embeddings', process.env.GENIENLP_EMBEDDINGS);
-        if (process.env.GENIENLP_DATABASE)
-            args.push('--database', process.env.GENIENLP_DATABASE);
-
-        this._child = child_process.spawn('genienlp', args, {
-            stdio: ['pipe', 'pipe', 'inherit']
-        });
-        this._child.on('error', (e) => {
-            this._failAll(e);
-            this._hadError = true;
-            this.emit('error', e);
-        });
-        this._child.on('exit', (code, signal) => {
-            //console.error(`Child exited with code ${code}, signal ${signal}`);
-
-            this._failAll(new Error(`Worker died`));
-            this._child = null;
-            this.emit('exit');
-        });
-
-        this._stream = new JsonDatagramSocket(this._child.stdout!, this._child.stdin!, 'utf8');
-
+        if (process.env.USE_KF_SERVING_INFERENCE) {
```

Comment:
We should probably use a parameter to Predictor or Worker instead of an environment variable, especially one that is not properly namespaced. I imagine almond-cloud will set the parameter if the almond-cloud config file says so, and other environments (eg eval step in kubeflow) will not set the parameter all.